source/assets/css/regions/_banner.scss

```diff
@@ -1,6 +1,10 @@
 .sl-r-banner {
   position: relative;
 
+  // add bottom padding to increase space
+  // above page titles (visible on mobile site)
+  padding-bottom: 3rem;
+
```

Comment:
this was the reason I wanted to see a screenshot. It's not scoped to a media query so it would be adding space to the desktop web view as well. I think if we can just scope this, too, we're all set.

Reply:
@jina here's a screenshot, the improvement is primarily targeted for mobile, but also adds some spacing for desktop as well. I plan to go back into this area and improve it further in a couple weeks.

Before/After:
![image](https://user-images.githubusercontent.com/2808046/97311016-2ca8fa80-1864-11eb-9221-4fd6a077de9a.png)